src/routes/userRoute.ts

```diff
@@ -42,8 +48,14 @@ router.get('/block', getBlockedUsers);
 router.post('/block/:id', block);
 router.delete('/block/:id', unblock);
 
+// Admin routes
+router.patch('/activate/:userId',isAdmin,activateUser);
+router.patch('/deactivate/:userId',isAdmin,deactivateUser);
+router.patch('/ban/:userId',isAdmin,banUser);
+router.get('/all-groups',isAdmin,getAllGroups);
+router.get('/',isAdmin, getAllUsers);
```

Comment:
Can you please change the ordering so that the final 3 routes are (in this order):
GET /:userId/stories
GET /:userId
GET /